index.js

```diff
@@ -100,7 +101,8 @@ module.exports = class Plugin {
 
   // The Broccoli builder calls plugin.__broccoliGetInfo__
   __broccoliGetInfo__(builderFeatures) {
```

Comment:
What happens when this is `null` (like it was before in the read-compat layer)? Does that just not happen anymore (since read-compat was fixed)?

Reply:
@rwjblue I don't believe it occurs any longer, but it does open a question:

Currently, if `__broccoliGetInfo__()` is invoked, we will error. The question is, should we:

a) error as it does now
b) assume the minimal builder feature set
c) assume the complete plugin capability feature set.

cc @rwjblue / @thoov  / @krisselden

Reply:
Some results off of github suggest that this will be called with no arguments sometimes also:

* https://github.com/broccolijs/broccoli-source/blob/e2290f7903fa73f8ea90f3600f7d81d8edb2a579/index.ts#L85
* https://github.com/ember-cli/broccoli-lint-eslint/blob/3bfc02489b6d31d32d3ba7f6b078f6a9ff9ba52d/lib/index.js#L24
* https://github.com/ember-cli/ember-cli/blob/ffb36c5a9028d9d7d66060f516a501ee284cd788/tests/unit/models/addon-test.js#L509

If this function is called with no args, it will error in `_checkBuilderFeatures`

Reply:
I don't think we can error (unless we bump major's, but not even sure then since things like broccoli-lint-eslint are calling it on its **inputs** which may or may not be in the major).

Reply:
I'd say we have to do `b`

Reply:
thats fair, so do we choose b) or c)

Reply:
done